### PR TITLE
Add license files

### PR DESCRIPTION
**Pull request by JohnTitor**
This repository is licensed under MIT/Apache-2.0 but I noticed there are no license files.

### CODE REVIEW

File: LICENSE-MIT [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Permission is hereby granted, free of charge, to any
```
Comment (jonas-schievink) [inline LICENSE-MIT]: The copyright notice is missing, is that intended?

Comment (JohnTitor) [reply LICENSE-MIT]: I refer to rustwide's license, it's also missing https://github.com/rust-lang/rustwide/blob/master/LICENSE-MIT
Should we have the notice?

Comment (Dylan-DPC) [reply LICENSE-MIT]: This is consistent with the other repos as well. e.g. [rust-lang](https://github.com/rust-lang/rust/blob/master/LICENSE-MIT)

Comment (jonas-schievink) [reply LICENSE-MIT]: Oh, okay, I guess it's fine then. I'm wondering how you're supposed to comply with it though.

**Comment by Dylan-DPC** [comment]
Merging this

**Pull request merged**